File: CarlaErik/UvA.SoftCon.Questionnaire/AST/Expressions/Boolean/GreaterThanExpression.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using UvA.SoftCon.Questionnaire.AST.Expressions.Numeric;

namespace UvA.SoftCon.Questionnaire.AST.Expressions.Boolean
{
    public class GreaterThanExpression : BinaryBooleanExpression<INumericExpression, INumericExpression>
    {
        public GreaterThanExpression(INumericExpression left, INumericExpression right)
            : base(left, right) { }

        public override bool Evaluate()
        {
            return Left.Evaluate() > Right.Evaluate();
        }
    }
}

File: KrasiMilan/QL/src/main/java/nl/uva/softwcons/ast/expression/binary/comparison/GreaterThanExpression.java
package nl.uva.softwcons.ast.expression.binary.comparison;

import nl.uva.softwcons.ast.expression.Expression;
import nl.uva.softwcons.ast.expression.binary.BinaryExpression;

public class GreaterThanExpression extends BinaryExpression {

    public GreaterThanExpression(Expression left, Expression right) {
        super(left, right);
    }

}
